### DISCUSSION TITLE
Script not working

### DISCUSSION THREAD

**Discussion by gltched-usr** [retypeapp/retype#504] [Q&A] [answered]
I want to add my own, selfhosted analytics added to my docs.
In normal HTML I would paste this into the `<head></head>`:
```
<script defer data-domain="SITE_NAME" src="LINK_TO_JS_FILE_ON_SERVER"></script>
```

Is it possible to integrate that into my retype doc page?

I tried pasting just the `<script>` on top of the .md document but my analytics page detected a page visit.

**Comment by TheDanniCraft** [comment, marked as answer]
I'm not sure what selfhosted analytics you use but if you use plausible(I asume that because the script tag you show looks like one generated by plausible) then just try to publish it to a domain.
Plausible ignores all page views from sites that are localhost(localhost or 127.0.0.1).
You can  quick check that by opening the console and look id you find `Ignoring Event: localhost` in the console.

I had the same issue, but got it fixed after deploying the page on to a real domain.

**Reply by DannyBen** [reply]
Some relevant discussion on Plausible:
https://github.com/plausible/analytics/discussions/445

And judging by the regex presented there, it looks like going to http://anything.localhost could work.

**Comment by geoffreymcgill** [comment]
Including site-wide custom JS or CSS is possible by adding your script to a `_includes/head.html` page. Whatever is in `head.html` will be automatically included into the `<head>` section of all generated pages.

More details on using `_includes/head.html` are available in issue https://github.com/retypeapp/retype/discussions/282#discussioncomment-2343005

Please see issue https://github.com/retypeapp/retype/discussions/272 for Plausible related thread.
